test/integration/suites/node-attestation/03-test-node-attestation

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Test node attestation api
+jointoken=`docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep jointoken`
+echo "Created Join Token" $jointoken
+
+svid1=`docker-compose exec -u 1001 -T spire-agent /opt/spire/conf/agent/node-attestation -testStep jointokenattest -tokenName $jointoken`
+if [[ $? -ne 0 ]];
+then
+	fail-now "Failed to do initial join token attestation"
+fi
+echo "Received initial SVID:" $svid1
+
+svid2=`docker-compose exec -u 1001 -T spire-agent /opt/spire/conf/agent/node-attestation -testStep renew -certificate "${svid1}"`
+if [[ $? -ne 0 ]];
+then
+	fail-now "Failed to do SVID renewal"
+fi
+echo "Received renewed SVID:" $svid2
+
+docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep ban -tokenName ${jointoken} 
+if [[ $? -ne 0 ]];
+then
+	fail-now "Failed to do initial join token attestation"
+fi
+echo "Agent banned"
+
+docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep renew -certificate "${svid2}"
+if [[ $? -eq 0 ]];
+then
```

Comment:
I believe that this will make the test to fail, since it will exit with a non-0 code here.

```suggestion
if docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep renew -certificate "${svid2}" ; then
```

Reply:
I will check

Reply:
Ok, I think either way is equivalent. We want the test to fail IF the exit value is zero (since renewing after banning should not work) :) .